packages/pwa-kit-create-app/assets/bootstrap/js/overrides/app/pages/home/index.jsx.hbs

```diff
@@ -51,7 +52,7 @@ const Home = () => {
     // and these objects are undefined on the client side.
     const {res} = useServerContext()
     if (res) {
-        res.set('Cache-Control', `s-maxage=${MAX_CACHE_AGE}`)
+        res.set('Cache-Control', `s-maxage=${MAX_CACHE_AGE}, stale-while-revalidate=31536000`)
```

Comment:
Should this be STALE_WHILE_REVALIDATE rather than the hard coded value?

Reply:
Yup.. fixed it.